concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md

```diff
@@ -76,6 +141,63 @@ To process a new **ColorBox** value, you need to handle the value change event.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            @value-changed="handleValueChange"
```

Comment:
Same notes here as in the [DateBox PR](https://github.com/DevExpress/devextreme-documentation/pull/362#pullrequestreview-362500110).